shell/app/common/utils/query-string.tsx

```diff
@@ -78,8 +78,14 @@ export function mergeSearch(obj?: { [prop: string]: any }, toString?: boolean, i
  * 合并参数并更新到当前url上
  * @param obj 需合并的query对象
  */
-export function updateSearch(search?: { [prop: string]: any }, opt?: Obj) {
-  goTo(`${location.pathname}?${mergeSearch(search, true, false, opt)}`);
+
+ interface IOpt {
+   gotoOption?: IGotoOptions;
+   [pro: string]: any;
+ }
+export function updateSearch(search?: { [prop: string]: any }, opt?: IOpt) {
+  const { gotoOption = {}, ...rest } = opt || { };
+  goTo(`${location.pathname}?${mergeSearch(search, true, false, rest)}`, gotoOption);
```

Comment:
will it impact other module which ref this function. Like
![image](https://user-images.githubusercontent.com/5175455/118246618-c79db500-b4d4-11eb-8ec8-07ac2d95bc03.png)